modules/core/src/test/java/org/apache/ignite/internal/metric/IoStatisticsSelfTest.java

```diff
@@ -190,9 +194,9 @@ private IgniteConfiguration getConfiguration(boolean isPersist) throws Exception
             DataStorageConfiguration dsCfg = new DataStorageConfiguration()
                 .setDefaultDataRegionConfiguration(
                     new DataRegionConfiguration()
-                        .setMaxSize(30L * 1024 * 1024)
-                        .setPersistenceEnabled(true))
-                .setWalMode(WALMode.LOG_ONLY);
+                        .setMaxSize(25 * MB)
```

Comment:
Looks like this is THE change that fixes test flakiness. Looks like 30Mb produces a flaky test, but 25Mb does not. Could you please add a comment explaining why this value of 25Mb was chosen (is there a calculation that demonstrates that a test will be stable with 25Mb, or it's empirically chosen)?

Reply:
Tried to fix it